fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -1859,6 +1860,14 @@ private String queryForApplicationTableName(final String datatable) {
         return applicationTableName;
     }
 
+    private void validateDatatable(final String datatable) {
+        final String sql = "SELECT application_table_name FROM x_registered_table where registered_table_name = ?";
```

Comment:
I would have used 
```
SELECT count(application_table_name) FROM x_registered_table where registered_table_name = ?
```
here

Reply:
@abraham-menyhart Would you please quickly change this? It would be much nicer with count and easier to read.

Reply:
Yes, already implemented the fix, and I agree with the suggestion. Testing it, but got some env issues.

Reply:
feel free to send the fix if your local env problems persist.